### ISSUE TITLE
Viewer Role

### ISSUE THREAD

**Issue by juanignaciosl** [CartoDB/cartodb#7716]
Feature doc: https://docs.google.com/document/d/1HNm7nNyfTH9z4zWiWZxeH3O73m_UAV5Ah3rjeqjsooQ

**Comment by javitonino** [comment]
Some quick question after playing with this (congrats! it works like a charm):
- The role is shown for all users, including those outside of organizations. Does this make sense? I mean, single users are always going to be Builders, so why show the role at all?
- I think it would be nice if viewer/builder tags had different colors.
- Builder/dataset views can still be accessed (manually, though URL) and are not blocked. The model/controller checks block all edition I tried though :dancer:

**Comment by juanignaciosl** [comment]
- I had mixed feelings about the user type tag being displayed for every user. Nevertheless, as it's still right (normal users are valid) and having different UI for normal and organization users make some things harded, I chose to keep them. Pinging @urbanphes here: should I remove the "builder" tag from the UI for normal (non-organization) users?
- That's a design decision, up to @urbanphes . Viewer is slightly lighter though.
- I guess that you mean the new views. I'll add a new issue about this, thanks!
